web/docs/understand/language/expressions.md

```diff
@@ -190,26 +195,26 @@ This makes it easier to search for IP addresses belonging to a specific subnet.
 Every [type](/docs/understand/data-model/type-system) has a corresponding
 value syntax in the expression language.
 
-Here is an over view of basic types:
-
-| Identifier | Description                             | Example Value
-| ---------- | --------------------------------------- | -------------
-| `none`     | Denotes an absent or invalid value      | `nil`
-| `bool`     | A boolean value                         | `true`, `false`
-| `int64`    | A 64-bit signed integer                 | `-42`, `+3`
-| `uint64`   | A 64-bit unsigned integer               | `0`, `42`
-| `double`   | A 64-bit double (IEEE 754)              | `-0.7`, `1.337`
-| `duration` | A time span (nanosecond granularity)    | `-3us`, `24h`
-| `time`     | A time point (nanosecond granularity)   | `now`, `1h ago`, `2020-01-01+10:42:00`
-| `string`   | A sequence of characters                | `"foo"`
-| `pattern`  | A regular expression                    | `/fo*.bar$/`, `/^\w{3}$/i`
-| `ip`       | An IPv4 or IPv6 address                 | `::1`, `10.0.0.1`, `2001:db8::`
-| `subnet`   | An IPv4 or IPv6 subnet                  | `::1/128`, `10.0.0.0/8`, `2001:db8::/32`
-
-Complex types:
-
-| Identifier  | Description                             | Example Data
-| ----------- | --------------------------------------- | ------------
-| `list`      | An ordered sequence of values where each element has type `T` | `[1, 2, 3]`, `[]`
-| `map`       | An associate array which maps keys to values | `{x -> a, y -> b, z -> c}`, `{}`
-| `record`    | a product type with one or more named fields | `<x: a, y: b, z: c>`, `<a, b, c>`, `<>`
+### Basic Types
```

Comment:
It's not strictly correct to call this types, as it refers to data, which while overlapping is a different thing. For example, there is no "none" type, but there is a "nil" value. There also is a "pattern" value without a corresponding type.

My suggestion is to just change Type to Literal and Example Value to Example in the column header, then it's correct. Also no need to distinguish between basic and complex here, as all values (except for nil) are stateful.